.github/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 1. Bug - Corrected Birmingham (EGBB) Holding Labels - thanks to @stuart5250 (Stuart Keaney)
 2. AIRAC (2014) - Updated EGHE Runway Headings - thanks to @AleksMax (Aleks Nieszczerzewski)
 3. Enhancement - Added EGBB/EGNX Buffer Zone - thanks to @AleksMax (Aleks Nieszczerzewski)
+4. Bug - Fixed location of Southend (EGMC) CTA-1 Airspace base label - thanks to @PLM1995 (Peter Mooney)
+5. Enhancement - Improved and updated Southend (EGMC) SMR - thanks to @PLM1995 (Peter Mooney)
```

Comment:
Because the changelog may be updated in-between of your pull request and the merge, it's usually best to place them at the top of the file with 'X.' as the number

Reply:
Changed, thanks for the tip!